libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -34,13 +39,16 @@ export class BookSearchComponent {
     this.store.dispatch(addToReadingList({ book }));
   }
 
+  onSubmitSearch() {
```

Comment:
What value does adding this additional method provide rather than having the template call searchBooks directly?

Reply:
I agree with you. adding a new method to call just another method won't add any value.

I addressed the above review comments and squashed the commit.